.github/workflows/vrt-baseline.yml

```
@@ -0,0 +1,106 @@
+        run: npm --workspace @jects/vrt run sb:build
+
+      # 로컬과 동일한 Docker 이미지로 baseline을 재생성한다.
+      # ref:run이 bitmaps_reference를 먼저 비우므로 삭제된 스토리의 남은 스냅샷은 제거된다.
+      - name: Regenerate baseline
+        run: npm --workspace @jects/vrt run vrt:reference
+        env:
+          CI: "true"
+
+      - name: Commit and push baseline
+        id: commit
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)